src/posthog-featureflags.ts

```diff
@@ -169,7 +169,9 @@ export class PostHogFeatureFlags {
                 // makes it through
                 this.$anon_distinct_id = undefined
 
-                this.receivedFeatureFlags(response as DecideResponse)
+                if (response && response.status === 200) {
```

Comment:
also worth checking: posthog-js is a bit nasty since there's 3 different places from which we call `/decide`, and I'm pretty sure this doesn't cover the case for where we bootstrap, and then find flags are obliterated on the first decide call, because that uses a different codepath (`new Decide()` something something..)

Reply:
Yeah, i moved the check directly into the parseDecide response function